content/en/docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -25,12 +25,38 @@ scheduling and evicting Pods.
 
 ## QoS classes
 
-When Kubernetes creates a Pod it assigns one of these QoS classes to the Pod:
+When Kubernetes creates a Pod it assigns one of these QoS classes to the Pod
+based on [resource limit and request](/docs/concepts/configuration/manage-resources-containers.md):
 
 * Guaranteed
+
+  Pods are considered high-priority and are guaranteed to not be killed until they exceed their limits. The scheduler assigns Guaranteed Pods only to nodes
+  which have enough resources to fulfill their resource requests.  
+  To be sure Pods will be included in Guaranteed Qos, every Container in the Pod must have resource(memory and CPU) limit and request, and they have to be same.  
+  The default CPU management policy of Kubernetes is [“None”](/docs/tasks/administer-cluster/cpu-management-policies/#none-policy). Under this policy Guaranteed Pods run in the shared CPU pool on a node.
+  To allocate exclusive CPU cores to Guaranteed Pods with [static](/docs/tasks/administer-cluster/cpu-management-policies/#static-policy) CPU management policy CPU requests should be an integer, Pods with fraction request of CPU cores will run in the shared CPU pool.    
+  Whenever a Kubelet reports [DiskPressure node condition](/docs/concepts/architecture/nodes/#condition), the scheduler stops scheduling any new Guaranteed Pods onto the node.  
+  And when an [eviction](/docs/tasks/administer-cluster/out-of-resource.md) happens on node, Guaranteed Pods have the lowest score ([oom_score_adj](/docs/tasks/administer-cluster/out-of-resource/#node-oom-behavior)) to be killed.
+
+
 * Burstable
+
+  Pods have some form of minimal resource guarantee, but can use more resources when available.  
+  To be sure a Pod will be included in Burstable QoS, at least one Contianer must have resource(memory or CPU) limit or request.  
```

Comment:
To be sure a Pod will be included in Burstable QoS, at least one container must have resource(memory or CPU) limit or request.